RHEL/7/input/xccdf/services/ldap.xml

```diff
@@ -26,13 +26,13 @@ network.</warning>
 <Rule id="ldap_client_start_tls" severity="medium">
 <title>Configure LDAP Client to Use TLS For All Transactions</title>
 <description>Configure LDAP to enforce TLS use. First, edit the file 
-<tt>/etc/pam_ldap.conf</tt>, and add or correct the following lines:
+<tt>/etc/openldap/ldap.conf</tt>, and add or correct the following lines:
```

Comment:
Based on https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/System_Administrators_Guide/ch-Directory_Servers.html#s2-ldap-introduction RHEL-7 instead of ```pam_ldap``` package provides ```nss-pam-ldap``` package. The proper location of the config file on RHEL-7 (when ```SSSD``` daemon is not used) is ```/etc/nslcd.conf``` -- this is the change performed by the ```authconfig-gtk``` command when ```LDAP``` was chosen as user account database, and ```[] Use TLS to encrypt connections``` option has been checked in:
```
# diff /etc/nslcd.conf.backup /etc/nslcd.conf
143a144,145
> ssl start_tls
> tls_cacertdir /etc/openldap/cacerts
```

For the future this check should be completely rewritten to recommend use of SSSD instead of ```nss-pam-ldapd``` (but that is future).